<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowui</class>
 <widget class="QWidget" name="mainwindowui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StudySpace</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <widget class="QWidget" name="mainArea" native="true">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>38</y>
     <width>870</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);</string>
   </property>
   <widget class="QLabel" name="homepageimage">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>214</y>
      <width>536</width>
      <height>272</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="listArea" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>38</y>
     <width>200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(225, 225, 225);</string>
   </property>
  </widget>
  <widget class="QWidget" name="infoArea" native="true">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>1</y>
     <width>870</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(126, 126, 126);</string>
   </property>
   <widget class="QToolButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(126, 126, 126);
border-radius: 32px;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="minimizeButton">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>0</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(126, 126, 126);
border-radius: 32px;</string>
    </property>
    <property name="text">
     <string>—</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="profileLabel">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>9</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="usernameLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>9</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#9e9e9e;&quot;&gt;USERNAME&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="logoArea" native="true">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>1</y>
     <width>201</width>
     <height>38</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(202, 202, 202);</string>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>201</width>
      <height>38</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(126, 126, 126);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; font-weight:700; font-style:italic;&quot;&gt;Study Space&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="funcOptionArea" native="true">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>48</width>
     <height>739</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(46, 46, 46);</string>
   </property>
   <widget class="QPushButton" name="chatOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>60</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>消息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="engLearnOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>180</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>英语</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contactsOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>120</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;
</string>
    </property>
    <property name="text">
     <string>好友</string>
    </property>
   </widget>
   <widget class="QPushButton" name="userinfoOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>690</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>我</string>
    </property>
   </widget>
   <widget class="QPushButton" name="feedBackOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>630</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>反馈</string>
    </property>
   </widget>
   <widget class="QPushButton" name="studyAidsOption">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>240</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius : 16px;</string>
    </property>
    <property name="text">
     <string>学业</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1121</width>
     <height>741</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 8px;
border-color: rgb(0, 0, 0);
border-style:solid;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>mainArea</zorder>
  <zorder>listArea</zorder>
  <zorder>infoArea</zorder>
  <zorder>logoArea</zorder>
  <zorder>funcOptionArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
